src/styles/blocks/card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  padding: 32px 16px 16px;
+  max-width: $card-width;
+  background: #fff;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $dark-blue;
+    margin: 40px 0 0;
+  }
+
+  &__product-code {
+    font-style: normal;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    margin-top: 4px;
+    color: $grey;
+  }
+
+  &__stars {
+    display: flex;
+    margin-top: 16px;
+    justify-content: space-between;
+    align-items: flex-end;
+  }
+
+  &__reviews {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $dark-blue;
+  }
+
+  &__price {
+    margin-top: 24px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+
+    &-name {
+      font-size: 12px;
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+      align-items: flex-end;
+      color: $grey;
+    }
+
+    &-num {
+      font-style: normal;
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+      color: $dark-blue;
+    }
+  }
+
+  &__button {
+    margin-top: 16px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    height: 40px;
```

Comment:
it's not a good practice to hardcode element with content inside. Use padding instead